### PR TITLE
feat: add new `debug-files print-sources` command

### PR DESCRIPTION
**Pull request by vaind**
This adds a new utility command to print sources included-in/referenced-by a debug file.

### CODE REVIEW

File: tests/integration/_cases/debug_files/debug_files-help.trycmd [hunk 1]
```diff
@@ -21,6 +21,7 @@ OPTIONS:
 SUBCOMMANDS:
     bundle-sources    Create a source bundle for a given debug information file
     check             Check the debug info file at a given path.
+    dump-sources      Print source files linked by the given debug info file.
```
Comment (vaind) [inline tests/integration/_cases/debug_files/debug_files-help.trycmd:24]: alternative name I was thinkig of: `show-sources`

File: tests/integration/debug_files/dump_sources.rs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+use crate::integration::register_test;
+
+#[test]
```
Comment (kamilogorek) [inline tests/integration/debug_files/dump_sources.rs:3]: Can you split it into two tests please?

File: tests/integration/_cases/debug_files/debug_files-help.trycmd (hunk 1 above)
Comment (kamilogorek) [reply tests/integration/_cases/debug_files/debug_files-help.trycmd:24]: I'd go with `print-sources`. `dump` is too easy to mistype with `dumb` and can be annoying 😅

**Review by kamilogorek — CHANGES_REQUESTED** [review]

File: tests/integration/_cases/debug_files/debug_files-help.trycmd (hunk 1 above)
Comment (vaind) [reply tests/integration/_cases/debug_files/debug_files-help.trycmd:24]: 👍

**Review by kamilogorek — APPROVED** [review]

**Pull request merged**